en/developer_guide/api.rst

```diff
@@ -0,0 +1,120 @@
+API
+===
+
+API access
+----------
+
+The connection to the API is based on the OAuth2 protocol.
+The OAuth2 protocol provides a client a secure access to an API on behalf of an user.
+
+A client is represented by a key/secret pair.
+In order to use the API, the client must request a token.
+
+The token is a string that refers to the client and optionally to the user. It is used to authenticate the client
+when he makes a request to the API.
+
+
+Client
+------
+
+The API clients can be managed in the administration section of the back office.
+
+The creation of a client is done through the add form.
+In the form the key/secret pair is automatically generated by the server.
+
+Moreover, in this form the client can be specified like trusted.
```

Comment:
You could rephrase it by ``The trusted parameter allows you to prevent connexion from a particular application to the api``